opcodes/arc-dis.c

```diff
@@ -1086,6 +1086,7 @@ print_insn_arc (bfd_vma memaddr,
   struct arc_operand_iterator iter;
   struct arc_disassemble_info *arc_infop;
   bool rpcl = false, rset = false;
+  bool code_section;
```

Comment:
```suggestion
  bool code_section = false;
```
nitpick; default init, even though it will always be set.

Reply:
I left it uninitialized intentionally to emphasize the fact that if `info->section == NULL` then `code_section == true` by default (it's the original logic of that code).

Reply:
I understand. But the point is that if the logic is changed by someone down the line and it no longer gets initialized in some cases, it defaults to something sane instead of becoming undefined.

Reply:
Up to you.

Reply:
For safety I would recommend to initialize as `true` and remove the `else` statement.

Reply:
Value of `code_section` depends on some non-trivial logic which should be evaluated separately. If we leave this variable initialized, then we encounter 2 issues:

 1. Duplicated initialization. First in the head of the function and the second one in `else` branch.
 2. Using of this variable before `if` will cause a warning and build fail. At the same time if this logic will be revisited and removed, then you will not be able to build this code without reconsidering usage of `code_section`. And if `code_section` is initialized, then a developer could forget to adjust it.

Anyway, there is no correct way to do it.